response_operations_ui/views/accounts.py

```diff
@@ -0,0 +1,164 @@
+    :param email: The email address to send to
+    """
+    url_safe_serializer = URLSafeSerializer(app.config['SECRET_KEY'])
+
+    response = uaa_controller.get_user_by_email(email)
+    if response is None:
+        return render_template('forgot-password-error.html')
+
+    if response['totalResults'] == 0:
+        internal_url = app.config['INTERNAL_WEBSITE_URL']
+        verification_url = f'{internal_url}/account/create-account/{token_decoder.generate_email_token(email)}'
+
+        logger.info('Sending create account email', verification_url=verification_url)
+
+        personalisation = {
+            'CREATE_ACCOUNT_URL': verification_url,
+            'EMAIL': email
+        }
+
+        try:
+            NotifyController().request_to_notify(email=email,
+                                                 template_name='request_create_account',
+                                                 personalisation=personalisation)
+        except NotifyError as e:
+            logger.error('Error sending create account request email to Notify Gateway', msg=e.description)
+            return render_template('request-new-account-error.html')
+
+        logger.info('Successfully sent create account request email', encoded_email=url_safe_serializer.dumps(email))
+    else:
+        logger.info('Requested account creation for email already in UAA',
+                    encoded_email=url_safe_serializer.dumps(email))
+        return render_template('request-new-account-exists.html', email=email)
+
+    return render_template('request-new-account-check-email.html', email=email)
+
+
+@account_bp.route('/resend_account_email_expired_token/<token>', methods=['GET'])
+def resend_account_email_expired_token(token):
+    email = token_decoder.decode_email_token(token)
+    return send_create_account_email(email)
+
+
+@account_bp.route('/create-account/<token>', methods=['GET'])
+def get_create_account(token, form_errors=None):
+    form = CreateAccountForm(request.form)
+
+    try:
+        duration = app.config['EMAIL_TOKEN_EXPIRY']
+        _ = token_decoder.decode_email_token(token, duration)
+    except SignatureExpired:
+        logger.warning('Token expired for Response Operations account creation', token=token)
+        return render_template('request-new-account-expired.html', token=token)
+    except (BadSignature, BadData):
+        logger.warning('Invalid token sent to Response Operations account creation', token=token)
+        return render_template('request-new-account-expired.html', token=token)
+
+    template_data = {
+        "error": {
+            "type": form_errors
+        },
+        'token': token
+    }
+    return render_template('create-new-account.html', form=form, data=template_data)
+
+
+@account_bp.route('/create-account/<token>', methods=['POST'])
+def post_create_account(token):
+    form = CreateAccountForm(request.form)
+
+    if not form.validate():
+        return get_create_account(token, form_errors=form.errors)
```

Comment:
This is the same thing as the other thing we talked about.  This shouldn't be using another routes function to do its job.  On the error, get all the bits you need and render the error template (you don't need the token verification functionality that's in the get_create_account function)